### PR TITLE
[Keyboard] Add Support to Ducky One2 SF

### PR DESCRIPTION
**Pull request by f7urry**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ducky/one2sf/1967st/chconf.h [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
```
[the remark this answers is not in the document]
Comment (f7urry) [reply keyboards/ducky/one2sf/1967st/chconf.h:17]: actualy i just fork from one2mini, and little changes on _1967st.h_ and add _ansi_ keymap, still cant figure out how matrix led working on MBI chipset

File: keyboards/ducky/one2sf/1967st/config.h [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x0416
+#define PRODUCT_ID      0x0123
```
Comment (drashna) [inline keyboards/ducky/one2sf/1967st/config.h:24]: https://www.the-sz.com/products/usbid/index.php?v=0x0416&p=0x0123&n=

This vendor appears to belong to nuvaton, but not actually an assigned product ID. 

I don't feel comfortable about using it, in this case, and would prefer an unused vendor ID.

**Review by drashna — CHANGES_REQUESTED** [review]
This needs a 'default' keymap.

File: keyboards/ducky/one2sf/1967st/config.h (hunk 2 above)
Comment (f7urry) [reply keyboards/ducky/one2sf/1967st/config.h:24]: so, i just put random vendor_id that unused by any vendor? or there are any regulation for that?

**Review by drashna — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Review by tzarc — APPROVED** [review]

**Pull request merged**